global.json

```diff
@@ -1,12 +1,12 @@
 {
   "tools": {
-    "dotnet": "2.1.401",
+    "dotnet": "2.1.602-preview-009587",
```

Comment:
@tmat is this the fix you expect for the breaking change https://github.com/dotnet/arcade/pull/2003?

Reply:
Updating to 2.1.503 should be good enough.

Reply:
Ah, true: that picks up NuGet 4.9. That peels this onion layer, thanks.